src/screens/slider/Slider.tsx

```diff
@@ -105,21 +140,25 @@ export const Slider: FC = () => {
             title={t(item.title)}
             text={t(item.text)}
             image={item.image}
+            isUserLoggedIn={isUserLoggedIn}
+            disableInitialAnimation={disableInitialAnimation || false}
```

Comment:
disableInitialAnimation param comes only from welcome screen, so it may be undefined and its || false, but I will add this props as optional in SliderContent, it may be a bit cleaner